misc/config_tools/data/sample_launch_scripts/apl-mrb/launch_user_vm.sh

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 kernel_version=$(uname -r)
-audio_module="/usr/lib/modules/$kernel_version/kernel/sound/soc/intel/boards/snd-soc-sst_bxt_sos_tdf8532.ko"
+audio_module="/usr/lib/modules/$kernel_version/kernel/sound/soc/intel/boards/snd-soc-sst_bxt_xxx_tdf8532.ko"
```

Comment:
Is this really a valid kernel module filename??

Reply:
no, this is not a valid kernel module, this part only used for specific audio. Just rename this kernel module.

Reply:
The snd-soc-sst_bxt_sos_tdf8532.ko also is not a valid kernel module name .

Reply:
Then why do you want to change it?...
BTW, does APL-MRB still needs to maintain?

Reply:
Delete the audio kernel module for specific board.